content/docs/dvclive/how-it-works.md

```diff
@@ -14,17 +14,19 @@ The contents of the directory will depend on the methods used:
 | [live.log_sklearn_plot](/doc/dvclive/api-reference/live/log_sklearn_plot) | `dvclive/plots/sklearn`    |
 | [live.make_report](/doc/dvclive/api-reference/live/make_report)           | `dvclive/report.{md/html}` |
 | [live.make_summary](/doc/dvclive/api-reference/live/make_summary)         | `dvclive/metrics.json`     |
+| [make_dvcyaml](/doc/dvclive/api-reference/live/end)                       | `dvclive/dvc.yaml`         |
```

Comment:
Not a blocker: we could probably make this a static method of `live` to keep a cleaner API.